Season-2/Level-3/code.py

```diff
@@ -40,37 +39,16 @@
 def index():
     if request.method == 'POST':
         planet = request.form.get('planet')
-        sanitized_planet = re.sub(r'[<>(){}[\]]', '', planet)
-
-        if 'script' in sanitized_planet.lower() :
-            return '<h2>Blocked</h2></p>'
-
-        elif sanitized_planet:
-            details = get_planet_info(sanitized_planet)
```

Comment:
The goal I mentioned in the [comment](https://github.com/skills/secure-code-game/pull/63/files#r1442365724) below, it's also achieved (hopefully!) with the chosen variable names, like `sanitized_planet` for example, making someone think that it's sanitized, hence safe!

Reply:
See my comment above.